website/api/models/VantaConnection.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * VantaConnection.js
+ *
+ * @description :: An organization who is a customer of Vanta.
+ * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
+ */
+
+module.exports = {
+
+  attributes: {
+
+    //  ╔═╗╦═╗╦╔╦╗╦╔╦╗╦╦  ╦╔═╗╔═╗
+    //  ╠═╝╠╦╝║║║║║ ║ ║╚╗╔╝║╣ ╚═╗
+    //  ╩  ╩╚═╩╩ ╩╩ ╩ ╩ ╚╝ ╚═╝╚═╝
+    emailAddress: {
+      description: 'The email address provided when this Vanta connection was created.',
+      type: 'string',
+      required: true,
+      isEmail: true,
+    },
+
+    vantaSourceId: {
+      description: 'The generated source ID for this Vanta Connection.',
+      type: 'string',
+      unique: true,
+      required: true,
+    },
+
+    fleetInstanceUrl: {
+      description: 'The full URL of the Fleet instance that will be connected to Vanta.',
```

Comment:
consider adding `example: 'https://fleet.example.com'` to indicate that there is the protocol and no trailing slash